src/layout/MobileCalculatorPage.jsx

```diff
@@ -13,6 +20,7 @@ import HOUSEHOLD_OUTPUT_TREE from "../pages/household/output/tree";
 import VariableSearch from "../pages/household/VariableSearch";
 import { ParameterSearch } from "../pages/PolicyPage.jsx";
 import PolicyRightSidebar from "../pages/policy/PolicyRightSidebar.jsx";
+import { impactKeys } from "../pages/policy/output/ImpactTypes.jsx";
```

Comment:
I couldn't add the comment at the correct line, but could you remove line 17 (`import {copySearchParams}...`) if it's no longer called?